@here/harp-atlas-tools/src/AtlasGenerator.ts

```diff
@@ -320,7 +320,7 @@ async function readAndProcessImage(
     logger.log(LogLevel.DEBUG, `Processing file: '${inFile}`);
     const imageDecoder = await imageFactory.loadImage(inFile);
     let imageOutput: ImageEncoder = imageDecoder.copy();
-    operations.forEach(imageProcess => {
+    operations.forEach((imageProcess) => {
```

Comment:
why is this needed now, it changes almost all files, is there a way to configure the new packages to work as before?

Reply:
Prettier v2 has now arrow-function-parentheses set to "always" by default, before it was "avoid".
https://prettier.io/docs/en/options.html#arrow-function-parentheses

I can add an additional rule in .prettierrc.json and remove those parentheses in case we want to stick with the previous style

Reply:
yes, please, lets stick to the previous formatting rules as far as possible.

Reply:
I added the rule to disable the parenthesis.
However, I couldn't do much about the space after the keyword 'function' on function expressions, e.g. `const foo = function () {..}`.
They don't seem to provide a way to suppress the rule - https://prettier.io/blog/2020/03/21/2.0.0.html#other-changes